strongbox-web-core/src/test/resources/etc/conf/strongbox.yaml

```diff
@@ -0,0 +1,331 @@
+configuration:
```

Comment:
How is this resource different from the one in the `strongbox-storage-api-resources` module?